### ISSUE TITLE
publish a preliminary version of udiff to PyPI 

### ISSUE THREAD

**Issue by sangyx** [Quansight-Labs/udiff#25]
We have implemented the basic functions. I think it's time to publish a preliminary version of udiff to PyPI. But I don't have the account. Could you publish it?

**Comment by hameerabbasi** [comment]
`udiff` isn't published on PyPI so far, so you should be able to publish this package fairly easily. I think it's about time we recognized your role in this project -- I'll add you to the collaborators. In addition, `unumpy` and `uarray` will need releases, I'll manage those.

**Comment by sangyx** [comment]
Hi, I encountered an error `Invalid value for requires_dist. Error: Can't have direct dependency: 'uarray @ git+https://github.com/Quansight-Labs/uarray@master#egg=uarray'` when I try to upload udiff to `pypi`. It seems that pypi does not allow direct URLs as the issue [#6301](https://github.com/pypa/pip/issues/6301) mentioned.

**Comment by hameerabbasi** [comment]
Yes, `unumpy` and `udiff` need releases, and we'll then need a PR to (temporarily) change the deps to point at the released versions, tag it, then release.
